### ISSUE TITLE
Applied Accessibility

### ISSUE THREAD

**Issue by rafaelgeronimo** [rafaelgeronimo/freeCodeCamp#4]
- [x] Add a Text Alternative to Images for Visually Impaired Accessibility
- [x] PassedKnow When Alt Text Should be Left Blank
- [x] PassedUse Headings to Show Hierarchical Relationships of Content
- [x] PassedJump Straight to the Content Using the main Element
- [x] PassedWrap Content in the article Element
- [x] PassedMake Screen Reader Navigation Easier with the header Landmark
- [x] PassedMake Screen Reader Navigation Easier with the nav Landmark
- [x] PassedMake Screen Reader Navigation Easier with the footer Landmark
- [x] PassedImprove Accessibility of Audio Content with the audio Element
- [x] PassedImprove Chart Accessibility with the figure Element
- [x] PassedImprove Form Field Accessibility with the label Element
- [x] PassedWrap Radio Buttons in a fieldset Element for Better Accessibility
- [x] PassedAdd an Accessible Date Picker
- [x] PassedStandardize Times with the HTML5 datetime Attribute
- [x] PassedMake Elements Only Visible to a Screen Reader by Using Custom CSS
- [x] PassedImprove Readability with High Contrast Text
- [x] PassedAvoid Colorblindness Issues by Using Sufficient Contrast
- [x] PassedAvoid Colorblindness Issues by Carefully Choosing Colors that Convey Information
- [x] PassedGive Links Meaning by Using Descriptive Link Text
- [x] PassedMake Links Navigatable with HTML Access Keys
- [x] PassedUse tabindex to Add Keyboard Focus to an Element
- [x] PassedUse tabindex to Specify the Order of Keyboard Focus for Several Elements
